### ISSUE TITLE
Renovate unchecks all "Ignored or Blocked" checkboxes in "Dependency Dashboard", but does not recreate/reopen PRs

### ISSUE THREAD

**Issue by janpio** [renovatebot/renovate#8719]
**What Renovate type, platform and version are you using?**

Github App

**Describe the bug**

We have a "Dependency Dashboard" issues that looks like this:

![image](https://user-images.githubusercontent.com/183673/108091343-ea5a3200-707b-11eb-93eb-265dc2bcd8b8.png)

I then check all the "Ignored or Blocked" checkboxes as I want all these PRs to be created again:

![image](https://user-images.githubusercontent.com/183673/108091485-0bbb1e00-707c-11eb-8fd3-8bd74a755f15.png)

But after Renovate runs and edits the issue, all checkboxes are unchecked, but the PRs are not opened again:

![image](https://user-images.githubusercontent.com/183673/108096823-b5e97480-7081-11eb-9d88-75d9d0c99010.png)

**Relevant debug logs**

https://app.renovatebot.com/dashboard#github/prisma/e2e-tests/303106059
https://app.renovatebot.com/dashboard#github/prisma/e2e-tests/303106060

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/master/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [ ] This is a really small bug, it does not need a reproduction (think small typo)
- [ ] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [x] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

Not sure that to look for in the logs, nothing jumped out to me to be related to these being unchecked but no action happening. Just don't know what to look for

**Comment by janpio** [comment]
Thanks, will try to recreate these another way then.

**Comment by rarkins** [comment]
Why did you need to check so many at once?

**Comment by janpio** [comment]
A week ago we had to quickly get rid of Renovate PRs in that branch to unblock GH Actions capacity to run for manual commits - and the quickest way was to just close all the open Renovate PRs. Now that I had some time again, I wanted to get Renovate running again for these dependencies.

I now manually reopened the PRs by clicking on the links, then re-creating the branches and reopening the PRs. Many were conflicted, so I then also requested rebases. Seems this is going well now.
